.travis.yml

```diff
@@ -1,14 +1,13 @@
 language: java
 sudo: false
 script: 
-  - git clone https://github.com/SupunArunoda/CPACEP.git
+  - ant build
+  - scripts/cpa.sh -spec config/specification/default.spc -config config/default.properties doc/examples/example.c -outputpath /tmp/original -stats
+  - cd ../CPACEP
   - mvn clean verify
```

Comment:
You shouldn't do both building and integration test in the same job.

Reply:
Hi Thomas,
Is it fine to use something like below to avoid this issue?

script: 
  - scripts/cpa.sh -spec config/specification/default.spc -config config/default.properties doc/examples/example.c -outputpath /tmp/original -stats
  - cd ../CPACEP
  - mvn clean verify
  
install:
  - cd ..
  - git clone https://github.com/sosy-lab/cpachecker.git
  - mkdir /tmp/original
  - cd cpachecker
  - ant build

Reply:
Hi Supun,
the execution of CPAchecker can be it's own job, so I would put it in a separate `script:`,
And it's still a problem that you have to assume in the `script` phase that you currently are in directory `cpachecker`.
Every stage should be able to assume that it's in the base directory, so I would add the (currently) `cd ../CPACEP`
to the end of the install stage.

Reply:
Hi Thomas,
Attached error occurs when using the above script.
![screenshot from 2018-07-01 22-44-04](https://user-images.githubusercontent.com/12041588/42136864-9541afb4-7d80-11e8-9ad3-a0d44a238bf3.png)

Reply:
Hi Supun, which above script? The one I told you to change or a new version of that?
Looks to me like cpachecker repo was cloned in `CPACEP`, but you go back one directory before executing `cpachecker/scripts/cpa.sh` .

Reply:
Hi Thomas,
Yes above script. Thats why I have created a separate folder outside the CPACEP because this has happened to me at the beginning too.

Reply:
That's because the path in `CPACEPConnectorTest` is wrong 😉 
`Path homePath = Paths.get(home, "build", "SupunArunoda", "cpachecker");`

Reply:
You mean Path homePath = Paths.get(home, "travis","build", "SupunArunoda", "cpachecker");

Reply:
Hm, what do you mean? 
I cited that statement from your code. 
It of course should be `SupunArunoda/CPACEP/cpachecker`.

That should be enough to get it working, but this will not work when changing the project name or for forks, and not when executing the test locally. 
So there, you should not use that absolute path at all.
There actually should be another review comment about that.